go/libraries/doltcore/sqle/alterschema_test.go

```diff
@@ -369,35 +363,32 @@ func TestDropColumnUsedByIndex(t *testing.T) {
 		colName        string
 		expectedIndex  bool
 		expectedSchema schema.Schema
-		expectedRows   []row.Row
 	}{
 		{
 			name:           "remove int",
 			colName:        "age",
 			expectedIndex:  true,
 			expectedSchema: dtestutils.RemoveColumnFromSchema(dtestutils.TypedSchema, dtestutils.AgeTag),
-			expectedRows:   dtestutils.TypedRows,
```

Comment:
Would be better to keep these assertions

Reply:
Maybe transform to SQL rows and run a select *

Reply:
I looked into it, but decided all these DDL tests are over-specified and don't need to mess with row data